src/handlers/canonicalize_issue_links.rs

```diff
@@ -15,10 +15,8 @@ use crate::{
     handlers::Context,
 };
 
-// Taken from https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue?quot#linking-a-pull-request-to-an-issue-using-a-keyword
 static LINKED_RE: LazyLock<Regex> = LazyLock::new(|| {
-    Regex::new("(?i)(?P<action>close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved)(?P<spaces>:? +)(?P<issue>#[0-9]+)")
-        .unwrap()
+    Regex::new("(?P<start> |^)(?P<issue>#[0-9]+)(?P<end>[^a-zA-Z0-9]|$)").unwrap()
```

Comment:
Maybe we can also just use `( |$)` for the `end` part? Otherwise things like `#1234č` will still get canonicalized, which is not linked by GitHub. Well, it's a very big edge-case anyway :) So up to you.

Reply:
Except that `#1.` is valid so we would need to include it, idk if it's worth the trouble.

Reply:
Ah, good point. Then `\b`?

Reply:
Sure, let's use that.